src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -61,13 +86,74 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     eventDoGet(response);
   }
 
+  /**
+   * Get user input.
+   * Generate directionsRequest from user input and parse optimized route.
+   * Create and store events in Datastore.
+   */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) 
       throws IOException {
     response.setContentType("application/json;");
 
+    Enumeration<String> params = request.getParameterNames();
+    String tripName = request.getParameter("inputTripName");
+    String origin = request.getParameter("inputDestination");
+    String startDate = request.getParameter("inputDayOfTravel");
+
+    // Print user input for now
+    response.getWriter().println("startDate: " + startDate);
+    response.getWriter().println("tripName: " + tripName);
+    response.getWriter().println("origin: " + origin);
+
+    // Save POIs to an array
+    List<String> pois = new ArrayList<>(); 
+    while (params.hasMoreElements()) {
+      String paramName = params.nextElement();
+      if (paramName.contains("poi")) {
+        String newPOI = request.getParameter(paramName);
+        pois.add(newPOI);
```

Comment:
Just as a heads up this logic will change when chris saves the POIs as a list rather than individual parameters.